.buildkite/pipelines/pmtile_synth_only.yml

```diff
@@ -20,7 +20,7 @@ steps:
   - $TEST --need_space 30G full_chip tile_array Tile_MemCore --steps synthesis --debug
 - wait: { continue_on_failure: true } # One step at a time + continue on failure
 
-- label: 'PE synth 23m'
+- label: '250MHz PE synth 12m'
```

Comment:
Note this label change has nothing to do with the larger merge, but I thought I'd take this opportunity to make it more useful / accurate.